runhttpd.sh

```diff
@@ -18,6 +18,17 @@ sed -i -e 's|\(^[[:space:]]*\)\(DocumentRoot\)\(.*\)|\1\2 "/shared/html"|' \
     -e 's|<Directory "/var/www/html">|<Directory "/shared/html">|' \
     -e 's|<Directory "/var/www">|<Directory "/shared">|' /etc/httpd/conf/httpd.conf
 
+# Remove log files from last deployment
+if [ -d "/shared/log/httpd" ]; then
+   rm -rf /shared/log/httpd/*
+else
+   mkdir -p /shared/log/httpd
+fi
+
+# Make logs available in shared mount
+ln -s /shared/log/httpd/access_log /var/log/httpd/access_log
```

Comment:
I *think* `ln -s` works on non-existing files, but I'd prefer we touched them anyway.

Reply:
I found it does work but added touch for consistency.